CONTRIBUTING.md

```diff
@@ -97,10 +101,10 @@ submitting a fix (even better :star2:).
 You can submit fixes to the documentation pages completely online without having to
 download and install anything:
 
-* On each documentation page, there should be an "Improve This Page" link at the very
+* On each documentation page, there should be an "Suggest edit" link at the very
   top.
```

Comment:
top (at the position of the GitHub logo).

Reply:
I find that's not very obvious for people who visit the website for the first time. Like I have to put my mouse to the GitHub logo to see the option for "Suggest edition". And the Verde website still shows" Improve this page" on the documentation page.

Reply:
Yeah, it's a bit hidden if you don't think to click on the GitHub icon. Thanks for catching this!

It will be at least consistent as soon as I merge the Verde docs update to the new theme.